src/PerfView/Utilities/SupportFiles.cs

```diff
@@ -178,6 +178,29 @@ public static string ExePath
             }
         }
         /// <summary>
+        /// The path to the executable for relaunching PerfView with administrator privileges.
```

Comment:
I would like to get rid of this special case.     SupportFiles is supposed to be generic code that knows nothing about PerfView.  

The comments in the ExePath property tell the story.   Over time this has morphed and now needs fixing.   Here is my proposed fix.

We should rename ExePath to 'MainAssemblyPath'  (it is defined as the assembly that contains the SupportFiles class).  

Then we refine ExePath to simply use 'GetEntryAssembly()', and use that to relaunch.